### PR TITLE
C7

### PR DESCRIPTION
**Pull request by JackRascal0**
Commit 7 by Jack

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,4 @@ Commit 3
 Commit 4  
 Commit 5  
 Commit 6  
+Commit 7 by Jack
```
Comment (ihsansul0) [inline README.md:8]: Jack, add 1 more commit! Make it Commit 8 by Jack!

**Review by ihsansul0 — CHANGES_REQUESTED** [review]
Need some changes.

File: README.md (hunk 1 above)
Comment (JackRascal0) [reply README.md]: All done Ihsan, you can check the code now, please tell me if there's anything wrong.

File: README.md [hunk 2]
```diff
@@ -5,3 +5,5 @@ Commit 3
 Commit 4  
 Commit 5  
 Commit 6  
+Commit 7 by Jack  
+Commit 8 by Jack
```
Comment (ihsansul0) [inline README.md:9]: Nice one, thank you for your support jack!

**Review by ihsansul0 — APPROVED** [review]
Changes accepted.

**Comment by ihsansul0** [comment]
Thank you for your contribution Jack.

**Pull request merged**